pkg/bucketeer/model/evaluation_detail.go

```diff
@@ -0,0 +1,60 @@
+package model
+
+type EvaluationDetail[T EvaluationValue] struct {
+	FeatureID      string
+	FeatureVersion int32
+	UserID         string
+	VariationID    string
+	Reason         EvaluationReason
+	Value          T
```

Comment:
For consistency with other SDKs, let's also add the variation name and rename the value field.
```suggestion
	VariationID    string
	VariationName    string
	VariationValue          T
	Reason         EvaluationReason
```